manifests/qpid.pp

```diff
@@ -6,9 +6,17 @@
 # @param wcache_page_size
 #   The size (in KB) of the pages in the write page cache
 #
+# @param service_ensure
+#   Specify if qpidd service should be running or stopped
+#
+# @param service_enable
+#   Enable qpidd service at boot
+#
 class katello::qpid (
   String $interface = 'lo',
   Integer[0, 5000] $wcache_page_size = 4,
+  Boolean service_enable = false,
+  Boolean service_ensure = false,
```

Comment:
What's the use case for splitting these at this level?

Also, I'd expect this to reuse the variable via params as a default. That way you can set it via globals to set the behavior.

Reply:
> What's the use case for splitting these at this level?

Mostly I'm just trying my best to follow patterns to ensure maximum flexibility as that always has seemed to be our modus operandi. I don't have an opinion which way I go with how many parameters or where they are exposed at. 

> Also, I'd expect this to reuse the variable via params as a default. That way you can set it via globals to set the behavior.

I still don't understand the params and global pattern in this module, so you'll need to explain to it to me like I am five (or just throw code example applied here at me).